tests/test_integration/test_user_resource.py

```diff
@@ -59,3 +59,8 @@ def test_get_sso_open_site_url_with_site_id(self):
         url = self.service.get_sso_open_site_url(
             self.user_id, site_id=self.site['id'])
         self.assertIn(self.site['id'], url)
+
+    def test_get_user_wsites(self):
+        user = self._create_user()
+        user_wsites = self.service.get_user_wsites(user['id'])
+        self.assertEqual(user_wsites, [])
```

Comment:
Make test with ws sites plz

Reply:
I suggest to do this test in separate test class (for example `TestYolaWSUser`), there you can use another creating ws user, which can create ws site https://github.com/yola/yolapy/blob/8809afa15f7903ac8b39ef0c7cdb99df3a9b32d1/tests/test_integration/test_user_resource.py#L12-L16

Reply:
And I think will be good to add removing user and site (for old tests too)

Reply:
Improved test. I don't think separate test case makes sense, because we're testing same resource in a single class.

Reply:
but you test it for is_ws user

Reply:
and let's add `tearDownClass` for old test case class with delting site and user

Reply:
And?

Reply:
You create another user in test

Reply:
Don't understand you, sorry. Test creates a user, and lists it's sites.

Reply:
your test doesn't uses anything from `setUpClass`. And if we will have another endpoints for ws users - it will better to `setUpClass` with creating ws user

Reply:
I wouldn't like to spend significant time on refactoring of these tests now. However, I agree they might be much better organized.
I suggest to file a ticket for this work and do it when we have more time.